components/monitoring/base/grafana-app.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: appstudio-workload-monitoring
+  name: grafana-serviceaccount
+  annotations:
+    argocd.argoproj.io/compare-options: IgnoreExtraneous
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: appstudio-workload-monitoring
+  name: ocp-injected-certs
+  labels:
+    config.openshift.io/inject-trusted-cabundle: "true"
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  namespace: appstudio-workload-monitoring
+  name: grafana-access
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: grafana-tls
+    kubernetes.io/tls-acme: "true"
+spec:
+  port:
+    targetPort: https
+  tls:
+    termination: reencrypt
+  to:
+    kind: Service
+    name: grafana-service
+    weight: 100
+  wildcardPolicy: None
+---
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  namespace: appstudio-workload-monitoring
+  name: grafana
+spec:
+  config:
+    log:
+      mode: "console"
+      level: "warn"
+    # auth.anonymous:
+    #   enabled: True
+    auth:
+      disable_login_form: False
+      disable_signout_menu: True
+    auth.basic:
+      enabled: False
+    auth.proxy:
+      enabled: True
+      enable_login_token: True
+      header_property: username
+      header_name: X-Forwarded-User
```

Comment:
How about allowing the users to have edit access to create their own dashboard?

Reply:
As already discussed on slack (IIRC), I don't think we need something like this for now.